pkg/rtc/participant.go

```diff
@@ -255,6 +238,59 @@ func NewParticipant(params ParticipantParams) (*ParticipantImpl, error) {
 	return p, nil
 }
 
+func (p *ParticipantImpl) createDataChannelForSubscriberAsPrimary() error {
+	primaryPC := p.subscriber.pc
+	ordered := true
+	// also create data channels for subs, this is for legacy clients that do not use subscriber
+	// as primary channel
```

Comment:
is this comment still correct? This function is called only for subscriber as primary.